### PR TITLE
YLP-343 Configure blip to use JWT in the Zendesk web widget

### PR DESCRIPTION
**Pull request by yvon-dblg**
New config variables / change one
- HELP_SCRIPT_URL
- HELP_PAGE_URL

### CODE REVIEW

File: packages/blip/app/pages/app/app.js [hunk 1]
```diff
@@ -183,23 +185,36 @@ export class AppComponent extends React.Component {
       }
     }
     if (config.HELP_LINK !== null && typeof window.zE === 'function') {
```
Comment (Nas84) [inline packages/blip/app/pages/app/app.js:187]: config.HELP_LINK should be rename as HELP_SCRIPT_URL

File: packages/blip/app/pages/app/app.js [hunk 2]
```diff
@@ -183,23 +185,36 @@ export class AppComponent extends React.Component {
...
-      if (this.props.authenticated) {
-        let name = this.props.user.profile.fullName;
-        let email = this.props.user.emails[0];
+      this.renderZenDeskWidget();
+    }
+  }
 
+  renderZenDeskWidget() {
```
Comment (Nas84) [inline packages/blip/app/pages/app/app.js:192]: it misses unit tests on this function (you could see the coverage report for more details)

**Review by Nas84 — CHANGES_REQUESTED** [review]

File: templates/zendesk.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            'x-tidepool-session-token': token
+          },
+          cache: 'no-cache',
+        }).then((response) => {
+          if (response.ok) {
+            const zendeskToken = response.headers.get('x-external-session-token');
+            console.info('Retrieve a new sso token for Zendesk');
+            return callback(zendeskToken);
+          }
+          console.warn('Zendesk callback SSO token: Missing token');
```
Comment (yvon-dblg) [inline templates/zendesk.js:20]: No call to callback in this case ?

File: packages/platform-client/user.js [hunk 4]
```diff
@@ -149,6 +149,10 @@ module.exports = function (common, config, deps) {
    * Destroy user session (in-memory and stored in browser)
    */
   function destroySession() {
+    if (config.zendeskSSOEnabled) {
+      log.info('Removing zendesk token');
+      window.sessionStorage.removeItem(ZDK_TOKEN_LOCAL_KEY);
```
Comment (yvon-dblg) [inline packages/platform-client/user.js:154]: ZDK_TOKEN_LOCAL_KEY is no longer defined.
Forgotten code to remove ?

**Review by Nas84 — APPROVED** [review]
Great, looks good (after the questions are addressed)

File: templates/zendesk.js (hunk 3 above)
Comment (ydrisR) [reply templates/zendesk.js:20]: I'm not sure what is best: not calling the callback seems to work well, at least it does not throw any error in the console. But if I call the callback with an invalid token (i.e. callback('invalid.token')) then it throws an exception...

**Pull request merged**